config/locales/interface/input_elements/en_demand_buildings.yml

```diff
@@ -296,3 +296,35 @@ en:
         taking the cold from the water and using it to cool one or more buildings.
         Because the cold stored in a lake goes down drastically during summer the
         year round capacity of deep lakes is small.
+    buildings_space_heater_heatpump_air_water_electricity_share:
+      title: Heat pump (air)
```

Comment:
I think this is a very clear name. We can change its siblings name from 'Gas-fired heat pump' to 'Heat pump (gas)' to match.
<img width="168" alt="Screenshot 2021-08-05 at 16 14 25" src="https://user-images.githubusercontent.com/14875123/128365257-1c22bb24-4db4-4022-bccc-e306e1011497.png">